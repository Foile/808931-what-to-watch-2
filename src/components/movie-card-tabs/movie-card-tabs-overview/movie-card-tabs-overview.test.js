import React from "react";
import renderer from "react-test-renderer";
import MovieCardTabsOverwiev from "./movie-card-tabs-overview";

it(`MovieCardTabsOverwiev correctly renders after relaunch`, () => {
  const tree = renderer
  .create(<MovieCardTabsOverwiev />)
  .toJSON();
  expect(tree).toMatchSnapshot();
});
