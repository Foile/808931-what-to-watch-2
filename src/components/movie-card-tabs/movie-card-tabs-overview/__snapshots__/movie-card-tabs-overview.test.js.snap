// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieCardTabsOverwiev correctly renders after relaunch 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    />
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Bad
      </span>
      <span
        className="movie-rating__count"
      >
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
        
         and other
      </strong>
    </p>
  </div>,
]
`;
